Note on String.p.includes case sensitivity (#11936)
mdn/content: files/en-us/web/javascript/reference/global_objects/string/includes/index.md

'Blue Whale'.includes('blue')  // returns false
```

> However, you emulate a case insensitive match by changing the case of the original string:

Reviewer: What about:
```suggestion
You can work around this constraint by changing the case of the string:
```
